apps/els_framework/src/els_framework.app.src

```diff
@@ -0,0 +1,31 @@
+{application, els_framework, [
+    {description, "Erlang Language Server Framework"},
+    {vsn, "0.1.0"},
+    {registered, []},
+    {applications, [
```

Comment:
Some of these apps seem specific to the LSP parts (eg `docsh`, `rebar3_format` or `tdiff`).

Reply:
I just put all external deps here, this needs sorting for sure